bentoml/cli/bento_service.py

```diff
@@ -221,9 +221,16 @@ def open_api_spec(bento=None):
         help="Run API server with micro-batch enabled",
         envvar='BENTOML_ENABLE_MICROBATCH',
     )
-    def serve(port, bento=None, enable_microbatch=False):
+    @click.option(
+        '--run-with-ngrok',
+        is_flag=True,
+        default=False,
+        help="(Beta) Run API server with ngrok enabled",
```

Comment:
Nit: `help="Use ngrok to relay traffic on a public endpoint to this API server on localhost"`